src/proto/deps/IdentityRpc.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+// protobuf vscode extension: https://marketplace.visualstudio.com/items?itemName=zxh404.vscode-proto3
+
+// Light-weight representation of a .NET System.Security.Claims.ClaimsIdentity object.
+// This is the same serialization as found in EasyAuth, and needs to be kept in sync with
```

Comment:
Nit: Add link to EasyAuth  and ClaimsIdentitySlim doc

Reply:
Added link to ClaimsIdentitySlim definition in Azure/azure-webjobs-sdk-extensions. I don't believe the EasyAuth definition is publicly available.